data/soapvalidator/Calendar/Appointments/CreateAppointmentRequest-Reminder.xml

```diff
@@ -142,7 +142,7 @@
         <t:response>
             <t:select path="//mail:GetMsgResponse/mail:m[@id='${appt1.invId}']">
                         <t:select path="//mail:comp">
-                                <t:select path="//mail:s" attr="d" match="${ICALTIME[${time.20071201120000.gmt}]}"/>
+                                <t:select path="//mail:s" attr="d" contains="${ICALTIME[${time.20071201120000.gmt}]}"/>
```

Comment:
Do you know why Z was added in start and end time? This testcase is not creating appointment with timezone.